translations/translations.json

```
@@ -5686,418 +5686,277 @@
   },
   "intro": {
     "de": {
-      "appDescription": "{{appName}} ist Dein digitaler Guide für Deutschland",
-      "search": "Suche",
-      "searchDescription": "Anhand von Schlagwörtern schnell die richtige Information finden",
-      "languageChange": "Sprachwechsel",
-      "languageChangeDescription": "Einfaches Umstellen der Sprache einzelner Seiten",
+      "welcome": "Willkommen bei {{appName}}",
+      "appDescription": "In {{appName}} findest du Antworten und Tipps in deiner Muttersprache. Zum Beispiel: Wo finde ich Hilfe? Wann darf ich arbeiten?",
```

:wrench: Please adjust the naming
```suggestion
      "welcomeDescription": "In {{appName}} findest du Antworten und Tipps in deiner Muttersprache. Zum Beispiel: Wo finde ich Hilfe? Wann darf ich arbeiten?",
```